From: Robert Schwebel <r.schwebel@pengutronix.de>
Date: Fri, 8 Jan 2010 17:22:45 +0100
Subject: [PATCH] buildmacros: use correct libtool commands

Signed-off-by: Robert Schwebel <r.schwebel@pengutronix.de>
[mkl: remove broken DESTDIR support]
Signed-off-by: Marc Kleine-Budde <mkl@pengutronix.de>
[mol: install .la not .lai for static libs]
Signed-off-by: Michael Olbrich <m.olbrich@pengutronix.de>
---
 include/buildmacros |    6 +++---
 1 files changed, 3 insertions(+), 3 deletions(-)

diff --git a/include/buildmacros b/include/buildmacros
index ab89182..d9b0734 100644
--- a/include/buildmacros
+++ b/include/buildmacros
@@ -55,10 +55,10 @@ LIBNAME = $(basename $(LTLIBRARY))
 LTOBJECTS = $(OBJECTS:.o=.lo)
 LTVERSION = $(LT_CURRENT):$(LT_REVISION):$(LT_AGE)
 
-LTLINK = $(LIBTOOL) --mode=link $(CC)
+LTLINK = $(LIBTOOL) --tag=CC --mode=link $(CC)
 LTEXEC = $(LIBTOOL) --mode=execute
 LTINSTALL = $(LIBTOOL) --mode=install $(INSTALL)
-LTCOMPILE = $(LIBTOOL) --mode=compile $(CCF)
+LTCOMPILE = $(LIBTOOL) --tag=CC --mode=compile $(CCF)
 
 ifeq ($(ENABLE_SHARED),yes)
 LTLDFLAGS += -rpath $(PKG_LIB_DIR)
@@ -99,7 +99,7 @@ endif
 INSTALL_LTLIB_STATIC = \
 	cd $(TOPDIR)/$(LIBNAME)/.libs; \
 	../$(INSTALL) -m 755 -d $(PKG_DEVLIB_DIR); \
-	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).lai $(PKG_DEVLIB_DIR)
+	../$(INSTALL) -m 644 -T old_lib $(LIBNAME).la $(PKG_DEVLIB_DIR)
 
 INSTALL_MAN = \
 	@for d in $(MAN_PAGES); do \
-- 
1.7.0.4

